src/mongocxx/options/index_view.cpp

```diff
@@ -25,7 +25,7 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
-index_view::index_view() : _max_time(), _write_concern() {}
+index_view::index_view() : _max_time(), _write_concern(), _commit_quorum() {}
```

Comment:
Forgot to do this during the commit_quorum support PR

Reply:
Acknowledged.